compiler/circle-mpqsolver/src/bisection/BisectionSolver.h

```diff
@@ -64,6 +64,11 @@ class BisectionSolver final : public MPQSolver
    */
   void algorithm(Algorithm algorithm);
 
+  /**
+   * @brief set path of visq_data
```

Comment:
this message doesn't seem to provide a help. (1) remove this (2) provide better message.